clouddriver-ecs/src/test/java/com/netflix/spinnaker/clouddriver/ecs/cache/ApplicationCacheClientTest.java

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright 2021 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * This file is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+ * CONDITIONS OF ANY KIND, either express or implied. See the License
+ * for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package com.netflix.spinnaker.clouddriver.ecs.cache;
+
+import static com.netflix.spinnaker.clouddriver.ecs.cache.Keys.Namespace.APPLICATIONS;
+import static org.junit.Assert.assertTrue;
+import static org.mockito.Mockito.when;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.netflix.spinnaker.cats.cache.DefaultCacheData;
+import com.netflix.spinnaker.clouddriver.ecs.cache.client.ApplicationCacheClient;
+import com.netflix.spinnaker.clouddriver.ecs.cache.model.Application;
+import com.netflix.spinnaker.clouddriver.ecs.provider.agent.ApplicationCachingAgent;
+import java.util.Collections;
+import java.util.Map;
+import org.junit.Test;
+import spock.lang.Subject;
+
+public class ApplicationCacheClientTest extends CommonCacheClient {
+  ObjectMapper mapper = new ObjectMapper();
+
+  @Subject
+  private final ApplicationCacheClient client = new ApplicationCacheClient(cacheView, mapper);
+
+  @Test
+  public void shouldConvert() {
```

Comment:
I think we need to update this test to validate that the app's relationships are also converted as expected

Reply:
We don't currently use the applicationcacheclient, so I can theoretically remove it completely

Reply:
moved logic into the clients and added tests!